src/components/VideoPlayerStatsForNerds.vue

```diff
@@ -138,13 +142,15 @@ const webrtcStats = new WebRTCStats({ getStatsInterval: 100 })
 function update(): void {
   framerateData.value.push(framerate)
   bitrateData.value.push(bitrate)
+  packetLostData.value.push(Math.min(packetLostDelta, maxPacketLost))
```

Comment:
Won't this put a roof on 10?

Reply:
yes. the reasoning here is that the number of lost packets is not as relevant as the fact that packets were lost, so if I left it "unbounded",  and we had 1000 packets lost between two messages, our ceiling would go to 1000. If we then lost 1 packet, it would be so small in the plot we wouldn't be able to see them.

![image](https://github.com/user-attachments/assets/3922a16b-62c0-44e9-97d2-a941b0bb7601)
In this image, for instance, I don't really care about how many packets were lost. I care that they were lost and we managed to recover.
Does that make sense?

TLDR: I put a low value so we can always notice EVERY packet lost

Reply:
Makes a lot of sense!